skimage/filters/thresholding.py

```diff
@@ -763,111 +761,98 @@ def threshold_multiotsu(image, nclass=3, nbins=255):
     >>> from skimage.color import label2rgb
     >>> from skimage import data
     >>> image = data.camera()
-    >>> thresh, _ = threshold_multiotsu(image)
+    >>> thresh = threshold_multiotsu(image)
     >>> regions = np.digitize(image, bins=thresh)
-    >>> sep_regions = label2rgb(regions)
+    >>> regions_colorized = label2rgb(regions)
     """
-    if image.shape[-1] in (3, 4):
-        raise TypeError("The input image seems to be RGB (shape: {0}. Please"
-                        "use a grayscale image.".format(image.shape))
-
-    if image.min() == image.max():
-        raise TypeError("The input image seems to have only one color: {0}."
-                        "Please use a grayscale image.".format(image.min()))
-
-    # check if nclass is between 2 and 5.
```

Comment:
the check of the correctness of nclass must be kept I think

Reply:
No, I don't want to merge this until nclass can be an arbitrary number until the number of bins.

Reply:
ok, I didn't get that. Sorry

Reply:
... Having said that, once that is implemented, I certainly wouldn't be opposed to a check that nclasses is strictly below the number of bins, once that's done. =)